docs/pvacbind/output_files.rst

```diff
@@ -67,8 +68,14 @@ all_epitopes.tsv and filtered.tsv Report Columns
      - Lowest ic50 binding affinity of all prediction algorithms used
    * - ``Best Score Method``
      - Prediction algorithm with the lowest ic50 binding affinity for this epitope
-   * - ``Individual Prediction Algorithm Scores`` (multiple)
-     - ic50 scores for the ``Epitope Seq`` for the individual prediction algorithms used
+   * - ``Median Percentile``
+     - Median ic50 binding affinity percentile rank of the epitope of all prediction algorithms used
```

Comment:
Since not all prediction algorithms have percentile output, maybe we could word this along the lines of "Median ic50 binding affinity percentile rank of the epitope of all prediction algorithms used with valid percentile output"?